code/__DEFINES/layers.dm

```diff
@@ -90,6 +90,9 @@
 #define GASFIRE_LAYER 5.05
 #define RIPPLE_LAYER 5.1
 
+///Layers for liquid effects
+#define LIQUID_EFFECT_LAYER 2.85
```

Comment:
No no no, sort this among the layers, as in, it goes between 2.7 and 2.9 or whatever. theres a massive list in this file that is sorted by number

Reply:
Ah gotcha.